src/content/get-started/advanced-commands-and-concepts.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Advanced Commands & Concepts for Developers
+description: Introduction to development concepts within Okteto
+sidebar_label: Advanced Commands and Concepts
+id: advanced-commands-and-concepts
+---
+
+import Image from '@theme/Image';
+
+## Advanced CLI Commands
+
+| Command          | Description                                                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| okteto namespace | Allows you to deal with namespaces (create, delete, use).                                                                   |
+| okteto exec      | Allows you to execute a command in your development container. `okteto exec [devName] -- COMMAND`                           |
+| okteto destroy   | Destroys a currently deployed Development Environment                                                                       |
+| okteto logs      | Used to view the logs of a Development Environment                                                                          |
+| okteto endpoints | Used to view your application’s HTTPS URLs automatically generated by Okteto                                                |
+| okteto doctor    | Generates a [doctor file](reference/okteto-cli.mdx#doctor) with the okteto logs for a given development container           |
```

Comment:
I would mention `okteto doctor` is useful for asking for Okteto support
Also, `okteto status` is very useful for troubleshooting (by the user) issues with syncthing